packages/date-picker/src/FilterDateRangePicker/components/Trigger/FilterTriggerButton/FilterTriggerButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { forwardRef } from "react"
+import { TriggerButtonBase, TriggerButtonBaseProps } from "../TriggerButtonBase"
+import styles from "./FilterTriggerButton.module.scss"
+
+export interface FilterTriggerButtonProps
+  extends Omit<TriggerButtonBaseProps, "children"> {
+  label: string
+  selectedValue?: string
+}
+
+export const FilterTriggerButton = forwardRef<
```

Comment:
Could we simplify the need for this abstraction?

Would something like this inside `FilterDateRangePicker.tsx` be enough:
```
<TriggerButton>
     {hasSelectedValue ? (
             <>
                 <span className={styles.hasSelectedValues}>{label}:&nbsp;</span>
                 <span>{selectedValue}</span>
             </>
      ) : (
        <span>{label}</span>
      )}
</TriggerButton>
```

So instead of the future state being about implementing a `XTriggerButton`, we just pass in different formatted labels depending on the filter's needs

Reply:
Yes that's in my next PR :) I copy pasted-ish from the FilterMultiSelect to just get it working first.